pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge PDF:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
+
+- Separate PDF into individual pages:
+
+`pdftk {{input.pdf}} burst`
+
+- Rotate third page by 90 degrees clockwise:
```

Comment:
Sure, I was just suggesting that if a space would work, it would be better to use such a syntax, as that would IMO make the command more understandable.

Since that's not an option, I'm thinking maybe swapping this command and the last one would allow one to decipher the intended syntax more easily. This would also adhere more closely to the [project guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines):
> Introduce options gradually, starting with the simplest command invocations, and using more complex examples progressively.